src/alert-prelude.c

```diff
@@ -510,6 +611,160 @@ static int PacketToDataV6(const Packet *p, const PacketAlert *pa, idmef_alert_t
     SCReturnInt(0);
 }
 
+#ifdef HAVE_LIBJANSSON
+/**
+ * \brief Convert JSON object to Prelude additional data with
+ * the right type of data.
+ *
```

Comment:
Comment on the param please and also about the algorithm used please.

Reply:
Done